witch_doctor.gemspec

```diff
@@ -18,14 +18,12 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["spec/**/*"]
 
-  s.add_dependency "rails", "~> 3.2"
+  s.add_development_dependency "rails", "~> 4.0.0"
```

Comment:
Ah, but strong_params is a requirement if (and only if!) you are using rails 3 - so your code breaks it (I think) :)

What I have put here currently "works", but I think it may be missing the s.add_dependency - which is only really important for people using this as a "public gem"